NakedJavaScript/mensa-projekt
# M27 essen in wochenansicht hinzufügen

# What has changed:
* You can now add a day meal in the index page, be aware that the day meal doesn't get displayed (this is another ticket).
# How to test:
* Go to the index page you should see the add day meal button, click it and select a meal
* You should get a message if the adding was successful but you won't see the meal added.
* To see if a meal was added, check the database under "tagesangebot".
